README.md

```diff
@@ -23,19 +23,23 @@ Set your preset in `.github/renovate.json`
 ## Rules
 
 > Description of the rules we need.
-> 🔗 means _included in the default config_.
+> :handshake: means _included in the default config_.
+> :sloth: means _included in the light config_.
 
-- 🔗 `github>SocialGouv/renovate-config//rules/group-deployment-tools` :
+- :handshake: `github>SocialGouv/renovate-config//rules/group-deployment-tools` :
   Group all the tools needed to deploy on the SocialGouv infra.  
-  Mainly the depedencies of your `.k8s/package.json`.
-
+  Mainly the depedencies of your `.k8s/package.json`.  
   See https://github.com/SocialGouv/kosko-charts/
 
-- 🔗 `github>SocialGouv/renovate-config//rules/node-lts-version` :
+- :sloth: `github>SocialGouv/renovate-config//rules/group-devdeployment-non-major` :
```

Comment:
double slash ?

Reply:
is a renovate config notation for path in a repo 
see https://docs.renovatebot.com/config-presets/#preset-hosting